pkg/apis/okctl.io/v1alpha1/cluster_v1alpha1.go

```diff
@@ -303,15 +306,50 @@ type ClusterDatabasesPostgres struct {
 
 // Validate the content of a postgres database declaration
 func (c ClusterDatabasesPostgres) Validate() error {
+	const minLengthDatabaseName int = 1
+
+	const maxLengthDatabaseName int = 60
```

Comment:
I'd move these to the const declaration on line 22. No need to declare these each time Validate() is called